src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -17,14 +13,29 @@
     public static final String COMMAND_WORD = "find";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-sensitive) and displays them as a list with index numbers.\n"
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
             + "Example: " + COMMAND_WORD + " alice bob charlie";
 
     private final Set<String> keywords;
 
     public FindCommand(Set<String> keywords) {
         this.keywords = keywords;
+        convertToLowerCase(keywords);
+    }
+
+    /**
+     * Converts given set of words to lower case for case insensitive find function
+     * @param words
+     */
+    private void convertToLowerCase(Set<String> words) {
+        String[] wordsArray = words.toArray(new String[0]);
+        for(int i = 0; i<wordsArray.length;i++){
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#layout) violation! One more more of these problems detected:
1. extra/missing spaces.
2. brace style problem.
3. missing braces.
4. name doesn't follow naming convention